libs/counterfactuals/src/lib/CounterfactualChart.tsx

```diff
@@ -779,6 +780,40 @@ export class CounterfactualChart extends React.PureComponent<
         trace.y = rawY;
       }
     }
+    if (
+      this.context.jointDataset.datasetMetaData !== undefined &&
+      this.context.jointDataset.datasetMetaData.featureMetaData !== undefined
```

Comment:
can this be shorten by `this.context.jointDataset.datasetMetaData?.featureMetaData !== undefined`? So we don't need line 784

Reply:
Done. Thanks for the suggestion.